CHANGELOG.md

```diff
@@ -2,11 +2,32 @@
 
 ## What's New
 
+* Initial proxy support in host.v1/host.v2
+
+## Proxy Support in host.v1/host.v2
+
+`host.v1` and `host.v2` configurations may now specify a proxy to use. 
+Currently only HTTP Connect proxies which don't require authentication are supported.
+
+**Example using `host.v1`**
+
+{
+    "address": "localhost",
```

Comment:
would the example make more sense if `address` (and perhaps also `proxy.address`) weren't "localhost", but remote/lan IPs?

Reply:
Done